### DISCUSSION TITLE
Visualization resets to Table each time input data changes

### DISCUSSION THREAD

**Discussion by xvcgreg** [enso-org/enso#5879] [Work In Progress - GUI] [answered]
Is that correct behavior for the visualization to change from let's say scatterplot (or anything other than table) to a table on change of the input data?

it can be observed at 1:56 and 2:22 mark:

https://user-images.githubusercontent.com/108271897/224111623-ef024b13-a5d6-4f0d-8c62-64ae1ff642b6.mp4

log: [2303091824_shareX.txt](https://github.com/enso-org/enso/files/10934516/2303091824_shareX.txt)

**Comment by farmaazon** [comment, marked as answer]
It is not. Once the user would pick a visualization from chooser, we should not change it.
